packages/atlas/src/views/global/YppLandingView/YppLandingView.styles.ts

```diff
@@ -18,6 +18,14 @@ export const imageShadow = css`
     drop-shadow(0 2.1945px 6.6501px rgb(0 0 0 / 0.0889)); */
 `
 
+export const Wrapper = styled.div`
+  [data-aos] {
+    &[data-aos][data-aos-easing='atlas-easing'] {
+      transition-timing-function: cubic-bezier(0.09, 0.43, 0.3, 0.99);
```

Comment:
How did you come up with this value? Is it Adam's suggestion?

Reply:
Yes, this comes from Adam. @toiletgranny maybe we should add it to design tokens?

Reply:
@rafalpawlow I really don't think we'll be using this easing value anywhere else, anytime soon. It's a really custom thing that works just for this LP. Would it make it any easier for you guys if this was a design token? I believe it's best if we keep them as simple as possible.

Reply:
Ok, I agree with @toiletgranny, I would opt to not add this to design tokens, not every value needs to be a token